util/reggen/countermeasure.py

```diff
@@ -31,6 +31,7 @@
     'DIFF',
     'REDUN',
     'SHADOW',
+    'MULTIRAIL'
```

Comment:
Missing comma here!

Reply:
Also, I think the only documentation for these names is in the (confidential) countermeasures sheet: is that right? If so, we should probably keep that in sync with this file.

Reply:
Thanks for spotting this. I've fixed that now.

Reply:
About the documentation, I've also just opened https://github.com/lowRISC/opentitan/issues/9985, which tracks documenting these properly.